build/webcharts.js

```diff
@@ -250,6 +250,30 @@
         this.events.onLayout.call(this);
     }
 
+    function addFontCSS() {
+        var context = this;
+
+        var styles = ['@import url(//fonts.googleapis.com/css?family=Open+Sans:400,300);'];
+
+        //Attach styles to DOM.
+        this.style = document.createElement('style');
+        console.log('okay?');
+
+        var request = new XMLHttpRequest();
+        request.open('HEAD', '//fonts.googleapis.com/css?family=Open+Sans:400,300');
+        request.send();
+
+        request.onreadystatechange = function() {
+            if (this.readyState == this.HEADERS_RECEIVED) {
+                console.log('gucci babies');
+                context.style.type = 'text/css';
+                context.style.innerHTML = styles.join('\n');
```

Comment:
also probs dont need the whole array join thing

Reply:
Let's remove the stray console log before merge .